scripts/linux/smokeTestHelper.sh

```diff
@@ -90,6 +144,76 @@ check-github-pmc-artifacts-similarity()
   fi
 }
 
+check-pmc-images-availability()
+{
+  # Check docker images availability in the PMC against all image tags.
+  #
+  # $1 - Github branch name
+
+  sourceBranchName=$1
+  taskDisplayNames=("Publish Edge Agent Manifest" "Publish Edge Hub Manifest" "Publish Temperature Sensor Manifest" "Publish Diagnostic Module Manifest")
+  for taskDisplayName in "${taskDisplayNames[@]}"
+  do
+    # Amalgam of test cases to test docker images in mcr
+    echo $'\n\n================================================\n\n'
+    check-pmc-image-tags-availability "$sourceBranchName" "$taskDisplayName"
+  done
+}
+
+check-pmc-image-tags-availability()
+{
+  # Check if the images pulled using docker image name from MCR resolves the to the same image SHA 
+  # which "Azure-IoT-Edge-Core Images Publish" pipeline claimed to publish to the MCR. 
+  #
+  # $1 - branch name
+  # $2 - task display name (use to reference the image module)
+  
+  branchName=$1
+  pipelineDisplayName=$2
+  echo "Checking images published for $branchName ($pipelineDisplayName)"
+
+  buildId=$(get-latest-image-publication-run "$branchName")
+  echo "BuildId: $buildId"
+
+  logs=$(get-build-logs-from-task "$buildId" "$pipelineDisplayName")
+
+  imageHashMap=$(get-image-sha-from-devops-logs "$logs")
+  echo "Checking the following images publication:"
+  echo "$imageHashMap"
+  echo ""
+
+  OLD_IFS=$IFS; 
+  IFS=' '
```

Comment:
What is this for? Name wasn't obvious to me.

Reply:
please let me know how can I improve the function description comments.

Reply:
The name IFS confused me, it isn't clear to me. Function name can be better. What is it?

Reply:
https://unix.stackexchange.com/questions/184863/what-is-the-meaning-of-ifs-n-in-bash-scripting

Reply:
Ah wonderful bash :). Add at a comment at your discretion.

Reply:
Why does the IFS change and not just stay what the default is?